corehq/apps/users/tasks.py

```diff
@@ -41,6 +42,13 @@ def bulk_upload_async(domain, user_specs, group_specs, location_specs):
         'messages': results
     }
 
+
+def run_case_post_save_signals(docs):
+    for doc in docs:
+        case = CommCareCase.wrap(doc)
+        case_post_save.send(CommCareCase, case=case)
```

Comment:
@dannyroberts @czue Do you forsee any problems with running all case post save signals on case deletion? The only other case post save action I could find (besides the reminders and sms ones which I know we need) was corehq.apps.receiverwrapper.create_case_repeat_records - is it ok if that's called with a deleted case?